app/routes/spin.js

```diff
@@ -0,0 +1,5 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model() { return this.get('store').findAll('spin-question') }
```

Comment:
This is an example of how ember-data would be used to query records from the API. In an actual dev environment (using the REST adapter), this would perform a GET request to `/spin_questions` and set the results as ember objects.